src/config.ts

```diff
@@ -15,5 +15,14 @@ export const ADYEN_LIVE_CLIENT_KEY = 'live_BQDOFBYTGZB3XKF62GBYSLPUJ4YW2TPL';
 // how often the live channel schedule is refetched in ms
 export const LIVE_CHANNELS_REFETCH_INTERVAL = 15 * 60_000;
 
-// OTT shared player
-export const DEFAULT_PLAYER_ID: string = import.meta.env.APP_DEFAULT_PLAYER ?? 'M4qoGvUk';
+// OTT shared player (for dev and prd api host)
```

Comment:
I would recommend combining the player and host constants into an environments hash like:
`const environments = {dev: {host: 'https://content-portal.jwplatform.com', player: 'GbLTFhkw'}, prd: {host: 'https://cdn.jwplayer.com', player: 'M4qoGvUk'}}` and it can be moved to a non-exported const in `loadAndValidateConfig` assuming the other changes are made as well.